qa/L0_perf_analyzer_example/test.sh

```diff
@@ -23,6 +23,6 @@
 
 : ${GRPC_ADDR:=${1:-"localhost:8001"}}
 
-perf_analyzer -m dali -i grpc -u $GRPC_ADDR -b 64 --input-data test_image --shape DALI_INPUT_0:`stat --printf="%s" test_image/DALI_INPUT_0` -f results.txt
+perf_analyzer -m dali -i grpc -u $GRPC_ADDR -b 64 --input-data test_image --shape DALI_INPUT_0:"$(stat --printf='%s' test_image/DALI_INPUT_0)" -f results.txt
 # Test if the perf_analyzer output is in a proper format and the measured times are in a reasonable range.
-[[ `tail -n1 results.txt` =~ ^1,[0-9]{3}\.[0-9],([0-9]{1,6},){10}[0-9]{1,6}$ ]] && echo "Output Correct"
+[[ "$(tail -n1 results.txt)" =~ ^1,[0-9]{1,4}\.[0-9],([0-9]{1,7},){10}[0-9]{1,7}$ ]] && echo "Output Correct"
```

Comment:
```suggestion
[[ "$(tail -n1 results.txt)" =~ ^1,[0-9]{1,}\.[0-9],([0-9]{1,},){10}[0-9]{1,}$ ]] && echo "Output Correct"
```
Maybe you should not put the upper bound of the number of occurrences?

Reply:
Makes sense, done!